migrations/20161021002706-create-topic.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+
+module.exports = {
+  up: function(queryInterface, Sequelize) {
+
+    const createUserTopicTable = () => {
+      return queryInterface.createTable('UserTopic', {
+        id: {
+          allowNull: false,
+          autoIncrement: true,
+          primaryKey: true,
+          type: Sequelize.INTEGER
+        },
+        UserId: {
+          allowNull: false,
+          type: Sequelize.INTEGER,
+        },
+        TopicId: {
+          allowNull: false,
+          type: Sequelize.INTEGER,
+        },
+        viewed: {
+          defaultValue: false,
+          allowNull: false,
+          type: Sequelize.BOOLEAN,
+        },
+        createdAt: {
+          allowNull: false,
+          type: Sequelize.DATE
+        },
+        updatedAt: {
+          allowNull: false,
+          type: Sequelize.DATE
+        }
+      }).then(() => {
+        return queryInterface.addIndex(
+          'UserTopic',
+          ['UserId', 'TopicId'],
+          {
+            indexName: 'UserTopicUniqueIndex',
+            indicesType: 'UNIQUE'
+          }
+        )
+      })
+    }
+
+
+    const createTopicsTable = () => {
```

Comment:
Was this created automatically?  Why not two separate migrations, one for each table (in the appropriate order to manage the foreign key fields)?